ltk/ltk.lisp

```diff
@@ -3445,6 +3447,44 @@ set y [winfo y ~a]
   (send-wish (format nil "~A read {~A} -shrink" (name p) filename))
   p)
 
+(defun list-of-numbers->str (list)
+  "Convert a list to string without the paranthesis."
+  (let ((s (format nil "~A" list)))
+    (subseq s 1 (1- (length s)))))
+
+(defun split-by-one-space (string)
+    "Returns a list of substrings of string divided by ONE space each."
+    (loop for i = 0 then (1+ j)
+          as j = (position #\Space string :start i)
+          collect (subseq string i j)
+          while j))
+
+(defun parse-list-of-integer (string)
+  (mapcar #'parse-integer (split-by-one-space string)))
+
+(defgeneric image-copy (p source-image &key from to shrink zoom))
+(defmethod image-copy ((p photo-image) source-image
+                                       &key (from '()) (to '()) (shrink nil) (zoom '()))
+  "Copy data from source image and resize if directed."
+  (format-wish "~A copy ~A ~A ~A ~A ~A"
+    (name p)
+    (name source-image)
+    (if (null from) ""
```

Comment:
my 2c: this indentation is not common, I was expecting this:

```lisp
(if (null from)
    ""
   (concatenate …))
```

(an the same with the other "if"s.

*thanks for sending a PR :star:*